content/applications/finance/accounting/reporting/data_inalterability.rst

```diff
@@ -24,27 +24,72 @@ would break the hash chain.
    bigger than the number of atoms in the known universe. This is why hashes are considered unique
    in practice.
 
-.. _data-inalterability/lock:
+.. _data-inalterability/inalterability_features:
 
-Lock entries with hash
-======================
+Inalterability Features
+=======================
+
+The Inalterability Features can be enabled by activating the :ref:`Secure Posted Entries with Hash
+<data-inalterability/restricted>` option on any journal or using the :ref:`Secure Entries Wizard
+<data-inalterability/wizard>`.
+
+- Two indicators are added to the journal entry form view.
+  They show whether the entry is secured or not.
+
+  - A lock icon next to the "Posted" state.
+  - A "Secured" checkbox in the "Other info" tab.
+
+- On journal entry list views a "Not Secured" filter is shown.
```

Comment:
```suggestion
- A :guilabel:`Not Secured` filter is available on journal entries' list views.
```

Reply:
It is now also available on the journal items' list views (so changed here too)